ftw/simplelayout/mapblock/profiles/default/types/ftw.simplelayout.MapBlock.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0"?>
+<object name="ftw.simplelayout.MapBlock"
+        meta_type="Dexterity FTI"
+        xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+        i18n:domain="ftw.simplelayout" >
+
+    <!-- Basic metadata -->
+    <property name="title" i18n:translate="">MapBlock</property>
+    <property name="description" i18n:translate="">Use this block to display a map</property>
```

Comment:
Could you remove this description?
I like the FTI description to always be empty because we often dont want any description and if there is a value here it is impossible to remove it just by using translations.